### PR TITLE
New Teleporter Pad System. Removing Shuttle 1 and 2

### PR DESCRIPTION
**Pull request by Enzo-Leon**
Updates the map with the new hyperpad system to allow rapid transit between the outpost and the space station. These are the pads recently added that will allow mechs and bluespace tier items to warp over safely. The old shuttles were removed and the code edited out to keep for posterity state.

Deck 1 Changes
deck_1_changes

Outpost Changes
outpost_changes

### CODE REVIEW

**Comment by Rykka-Stormheart** [comment]
This is super convenient and I approve, but, I am gonna miss the shuttles.

No way we can have a singular shuttle as emergency transit back and forth even if there's power loss that makes the teleporters nonfunctional?

Like, a detached hangar to the side of the base would be fantastic, imo.

**Comment by BlackMajor** [comment]
I'm biased towards this because I'd always circumvent the shuttles with teleporters lmao.


> No way we can have a singular shuttle as emergency transit back and forth even if there's power loss that makes the teleporters nonfunctional?
The Hyper Pads do not use power.

**Comment by Kazzc** [comment]
Nnggghh. . . Don't get me wrong, cutting downtime for players to go between the planet and station is good. But we have a very nice shuttle system that personally really adds to the scene and scenery of the game. One of the things I love about science is its ability to improve the station, make things faster, and save time. One of those things that science can do is build teleport pads.

I will budge in at the idea of making the shuttles faster, can take between 5-10 minutes to travel. Maybe dum it down to 1-3 minutes of travel time. (And get rid of its automated transport messages)

But personally, I will say no at the hyper pads.

**Comment by Enzo-Leon** [comment]
> Nnggghh. . . Don't get me wrong, cutting downtime for players to go between the planet and station is good. But we have a very nice shuttle system that personally really adds to the scene and scenery of the game. One of the things I love about science is its ability to improve the station, make things faster, and save time. One of those things that science can do is build teleport pads.
> 
> I will budge in at the idea of making the shuttles faster, can take between 5-10 minutes to travel. Maybe dum it down to 1-3 minutes of travel time. (And get rid of its automated transport messages)
> 
> But personally, I will say no at the hyper pads.

Main thing here is that.. with overmaps I plan to work on adding a LOT more stuff for folks, including a medi-vac ship that can go between places, but the SIF to Outpost is also reeking with lag and constant stuttering. This would also solve that issue I feel, and is a single part of a multi-part feature update I wanna help get for the server.

**Comment by BlackMajor** [comment]
> Main thing here is that.. with overmaps I plan to work on adding a LOT more stuff for folks, including a medi-vac ship that can go between places, but the SIF to Outpost is also reeking with lag and constant stuttering. This would also solve that issue I feel, and is a single part of a multi-part feature update I wanna help get for the server.

Gonna vocate my vouch here that Enzo a bean who'll probably actually follow through on that.

**Comment by Kazzc** [comment]
> > Nnggghh. . . Don't get me wrong, cutting downtime for players to go between the planet and station is good. But we have a very nice shuttle system that personally really adds to the scene and scenery of the game. One of the things I love about science is its ability to improve the station, make things faster, and save time. One of those things that science can do is build teleport pads.
> > I will budge in at the idea of making the shuttles faster, can take between 5-10 minutes to travel. Maybe dum it down to 1-3 minutes of travel time. (And get rid of its automated transport messages)
> > But personally, I will say no at the hyper pads.
> 
> Main thing here is that.. with overmaps I plan to work on adding a LOT more stuff for folks, including a medi-vac ship that can go between places, but the SIF to Outpost is also reeking with lag and constant stuttering. This would also solve that issue I feel, and is a single part of a multi-part feature update I wanna help get for the server.

I mean, sure. Dope that you have plans on adding more exploration content but not sure how is that related on building a hyper-pad that players can't build or dismantle, that will the shuttle system obsolete and that is not a pretty aesthetic compared to shuttling between two points, and science already being able to improve the time between those two points by being science.

**Comment by Enzo-Leon** [comment]
Normally I am 100% for giving people stuff to do, as I know ya all hear me chatter about that and enjoy the chaos/events that go on the station. However it is more of a chore when it's a _need this done every shift thing in order to actually make the work of other departments not slog and slow them down_.

Regarding the overmaps changes, I would be adding the new/updated shuttles to land about on SIF itself with new locations, and the ability for different types of ships to land. I left the south-east hanger empty for now as I plan on making that more into a larger room for more usage of civilian/medical/security response vessels as we move away from the carrier that was more or less abandoned from what I was told.

**Comment by Razgriz1032** [comment]
No. Just halve the shuttle times.

**Comment by BlackMajor** [comment]
I'm gonna state a quick opinion on this that iffen the playerbase likes it more then the shuttles then I'd kinda start nudging for this to be merged.

Y'could always test merge and gather feedback on it.

**Comment by GhostActual** [comment]
Darn, and I just got this all nice and set up for y'alls shuttles

sc_shuttle-n

**Comment by BlackMajor** [comment]
> Darn, and I just got this all nice and set up for y'alls shuttles

Use it anyway for the like, three other shuttles that use that template

**Comment by WTFIsmyname** [comment]
Shuttles are fucked to shit.
They take fucking ages
They literally lag just standing on the fucking things
Everyone tries to replace them as soon as they can
Literally who wants these things anymore.

File: maps/southern_cross/southern_cross-1.dmm [hunk 1]
```diff
@@ -1042,7 +986,7 @@
 "aCp" = (/obj/machinery/atmospherics/valve/digital/open{dir = 4},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/turf/simulated/floor/plating,/area/maintenance/firstdeck/foreport)
 "aCq" = (/obj/machinery/atmospherics/pipe/simple/hidden/universal{dir = 4},/obj/structure/cable{d1 = 1; d2 = 8; icon_state = "1-8"},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/structure/catwalk,/turf/simulated/floor/plating,/area/maintenance/firstdeck/foreport)
 "aCr" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/effect/floor_decal/steeldecal/steel_decals4{dir = 8},/obj/effect/floor_decal/steeldecal/steel_decals4{dir = 5},/turf/simulated/floor/tiled,/area/hallway/primary/firstdeck/fpcenter)
-"aCs" = (/obj/item/weapon/storage/toolbox/syndicate,/obj/structure/closet,/obj/random/contraband,/obj/random/maintenance/clean,/obj/random/maintenance/cargo,/turf/simulated/floor,/area/maintenance/firstdeck/foreport)
+"aCs" = (/obj/machinery/power/quantumpad,/turf/space,/area/space)
```
Comment (harry3124) [inline maps/southern_cross/southern_cross-1.dmm:989]: Quantumpad in space. Something that may need an adjustment later on.

**Pull request merged**